app/services/mandataris.js

```diff
@@ -101,4 +106,64 @@ export default class MandatarisService extends Service {
     });
     await newLidmaatschap.save();
   }
+
+  async updateOldLidmaatschap(mandataris) {
+    const oldLidmaatschap = await mandataris.heeftLidmaatschap;
+    if (!oldLidmaatschap) {
+      return;
+    }
+    let oldTijdsinterval = await oldLidmaatschap.lidGedurende;
+
+    if (!oldTijdsinterval) {
+      // old membership instances don't necessarily have a tijdsinterval
+      oldTijdsinterval = this.store.createRecord('tijdsinterval', {
+        begin: mandataris.start,
+        einde: moment(new Date()),
+      });
+      await oldTijdsinterval.save();
+      oldLidmaatschap.lidGedurende = oldTijdsinterval;
+      await oldLidmaatschap.save();
+    }
+    oldTijdsinterval.einde = this.date;
+
+    await oldTijdsinterval.save();
+  }
+
+  async createNewProps(mandataris, overwrites) {
+    return {
+      rangorde: overwrites.rangorde ?? mandataris.rangorde,
+      start: overwrites.start ?? mandataris.start,
+      einde: overwrites.einde ?? mandataris.einde,
+      bekleedt: overwrites.bekleedt ?? (await mandataris.bekleedt),
+      isBestuurlijkeAliasVan:
+        overwrites.isBestuurlijkeAliasVan ??
+        (await mandataris.isBestuurlijkeAliasVan),
+      beleidsdomein:
+        overwrites.beleidsdomein ?? (await mandataris.beleidsdomein).slice(),
+      status: overwrites.status ?? (await mandataris.status),
+      publicationStatus:
+        overwrites.publicationStatus ?? (await mandataris.publicationStatus),
+    };
+  }
+
+  async isMandatarisActive(mandataris) {
+    // assuming that there always be a folded mandataris for the mandataris
+    const foldedMandataris = (await fold([mandataris])).at(0);
+    const now = moment();
+    const todayIsInBetweenPeriod =
+      moment(foldedMandataris.foldedStart).isBefore(now) &&
+      (!foldedMandataris.foldedEnd ||
+        moment(foldedMandataris.foldedEnd).isAfter(now));
+
+    if (!todayIsInBetweenPeriod) {
+      return false;
+    }
+
+    const status = await foldedMandataris.mandataris.status;
+    return (
+      status &&
+      status.uri !== MANDATARIS_VERHINDERD_STATE &&
```

Comment:
@bfidlers  you where not sure about this?

Reply:
It is indeed only beeindigd, but this is fixed in the next version